app/models/user.rb

```diff
@@ -1,9 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Well done adding the devise, but as part of the requirements, you are supposed to ask for confirmation when signing in. Kindly add `:confirmable`